lib/tasks/maintenance.rake

```diff
@@ -43,4 +43,38 @@ namespace :maintenance do
       from_stage.destroy! if delete_empty && from_stage.deploy_groups.empty?
     end
   end
+
+  # FROM and TO handles multiple deploy groups to not mess with projects that have each deploy group in a single stage.
```

Comment:
wut ?
... we want to mess with these too

Reply:
copy pasta from other task